src/content/admin/cloud-credentials/gcp.mdx

```diff
@@ -38,29 +48,14 @@ Within the pool, let's register the kubernetes cluster as an identity provider.
     --workload-identity-pool=${POOL_ID} \
     --display-name=$(WORKLOAD_PROVIDER_NAME) \
     --attribute-mapping="google.subject=assertion.sub" \
-    --issuer-uri=https://container.googleapis.com/v1/projects/$(PROJECT)/locations/us-east2/clusters/development \
+    --issuer-uri="${OIDC_ENDPOINT}" \
     --allowed-audiences=${AUDIENCE}
 ```
 
-Okteto displays the OIDC endpoint of your cluster in the Admin -> General view of the Dashboard:
-
-  <p align="center">
-     <Image
-       src={require("@site/static/img/amazon-ecr-oidc-config.png").default}
-       alt="OIDC configuration in admin general view"
-     />
-   </p>
-
 Traditionally, `AUDIENCE` is the client id of the requester. You'll only exchange token for these audiences. It is the `aud` field of the JWT payload.
 
 We recommend creating a different/dedicated audience for each Okteto instance and region used, even if you use the same identity provider:
 
-```
```

Comment:
It seems very odd to show AWS nomenclature when both the displayed cluster and the cloud provider to be federated are in GCP.